helm/profiles.yaml

```diff
@@ -20,6 +20,24 @@ daskhub:
                   values:
                   - gpu
 
+        # Spark -----------------------------------------------------
+        - display_name: "PySpark"
+          default: "True"
+          description: '4 cores, 32 GiB of memory. <a href="https://github.com/pangeo-data/pangeo-docker-images" target="_blank">Pangeo Notebook</a> environment powered by <a href="https://rasterframes.io/">Raster Frames</a>, <a href="http://geotrellis.io/">GeoTrellis</a> and <a href="https://spark.apache.org/">Apache Spark</a>.'
+          kubespawner_override:
+            image: "${pyspark_image}"
```

Comment:
Note: the only difference between PySpark and Python images is in the amount of the underlying deps.